packages/amplify-category-notifications/lib/channel-APNS.js

```diff
@@ -156,11 +156,11 @@ function pull(context, pinpointApp) {
     ApplicationId: pinpointApp.Id,
   };
   spinner.start(`Pulling ${channelName} Channel.`);
-  return new Promise((resolve, reject) => {
+  return new Promise((resolve, reject) => {//eslint-disable-line
```

Comment:
why eslint disable line? If we keep on disabling linter where ever it fails to fix automatically there is no value in using it at all. We should avoid doing this

Reply:
A new Promise looks much better with both (resolve, reject) in it, but we don't need to reject in this, and eslint complains unused parameter.

Reply:
👍